docs/manual/en/manual.txt

```diff
@@ -0,0 +1,258 @@
+
+.Chord played on the "Harpsichord" instrument, without arpeggiation
+audio::../resources/arpeggio-no.opus[]
+
+.Chord played on the "Harpsichord" instrument, with arpeggiation
+audio::../resources/arpeggio-yes.opus[]
+
+Arpeggiation indicates insufficient polyphony. To prevent arpeggiation, the number of chips or 4op voices should be increased.
+
+=== Emulators
+
+One of several YMF262 chip emulators may be selected. Emulators may be switched dynamically.
+
+During the time required for switching emulators, the sound will be muted, and the MIDI stream will be ignored. This will not interfere with real-time operation.
+
+.Emulators
+image::../resources/emulators.png[200,200]
+
+- DOSBox OPL: This emulator is a balanced choice, producing good fidelity sound while using a very reasonable quantity of CPU resources.
+- Nuked OPL: This emulator offers excellent fidelity, aiming to provide nearly cycle-exact emulation. However, CPU usage is significant, so it is only recommended with a small number of chips, or for non-realtime rendering.
+
+=== Operators
+
+The four operators, whether functioning as modulator or carrier, provide the same parameters.
+
+.An operator
+image::../resources/operator.png[270,270]
+
+==== Waveforms
+
+The first parameter is the oscillator waveform, located in the lower left corner. One of 8 waveforms may be sellected.
+
+
+. sine
+. half-sine
+. absolute sine
+. quarter-sine, also known as pulse sine
+. alternating sine
+. camel sine, also known as absolute alternating sine
+. square
+. logarithmic sawtooth, also known as derived square
+
+==== Envelopes
+
+The envelope generator is controlled by four dials labelled "ADSR".
+The envelope controls the amplitude (volume) of an oscillator with respect to time.
+
+.ADSR Envelope
+image::../resources/adsr.svg[250,250]
+
+The attack (A) and decay (D) parameters control the durations of those envelope stages.
+
+The sustain (S) parameter controls the volume level held after A + D has elapsed, and as long as the key remains pressed.
+However, if the "Sustain" flag is disabled, there is no sustain stage, and the envelope effectively becomes an "ADR".
+
+The release (R) parameter comes after sustain, and similarly to "A" and "D", specifies the duration of the release stage.
+
+==== Flags
+
+The "Tremolo" and "Vibrato" flags apply small modulations to the operator, affecting volume and frequency respectively.
+
+"Sustain" enables or disables the "S" stage of the envelope. Refer to the explanation on envelopes.
+
+"Key scaling" automatically shortens the duration of the envelope as the pitch of the note increases.
+
+==== Levels
+
+The output level "Lv" is the global volume of the operator.
+Note that there is no "0" setting; even at the lowest level, the operator can be heard faintly.
+
+The frequency multiplier "F*" controls the base frequency of the operator, in octaves.
+
+The key scale level "Ksl" determines the extent to which amplitude is attenuated as pitch increases.
```

Comment:
Should I specify the actual KSL values, since they can be a bit surprising? ie

- 0 -> 0
- 2 -> 1.5 db/oct
- 1 -> 3 db/oct
- 3 -> 6 db/oct

Reply:
Yes, reminding me, a slider is not good for Ksl and I should change this.